lua/hydra/health.lua

```diff
@@ -1,31 +1,31 @@
 local M = {}
 
-local start = vim.health.start or vim.health.report_start
-local ok = vim.health.ok or vim.health.report_ok
-local warn = vim.health.warn or vim.health.report_warn
+local start = vim.health.start or vim.health.start
+local ok = vim.health.ok or vim.health.ok
+local warn = vim.health.warn or vim.health.warn
```

Comment:
This line makes me immediately distrust the rest of the PR. was this AI generated?

Reply:
My apologies. I have auto format set on my Nvim and didn't think about what that would mean when updating any of the code. 

None of it is AI generated, I just went through and updated the native functions that were flagging as deprecated because it was causing my config to fail when requiring Hydra.

Reply:
I will disable my auto format and redo the modifications when I have a chance to hopefully make it less crazy looking. My apologies for the confusion.